languages/cpp/exercises/concept/basics/.docs/after.md

```diff
@@ -0,0 +1,70 @@
+C++ is a statically-typed language, which means that everything has a type at compile-time. Assigning a value to a name is referred to as defining a variable. A variable can be defined either by explicitly specifying its type, or by using the [`auto` keyword][auto] to have the C++ compiler infer its type based on the assigned value.
+
+```cpp
+int explicitVar = 10; // Explicitly typed
+auto implicitVar = 10; // Implicitly typed
+```
+
+Updating a variable's value is done through the [`=` operator][assignment-operators]. Once defined, a variable's type can never change.
+
+```cpp
+auto count = 1; // Assign initial value
+count = 2;     // Update to new value
+
+// Compiler error when assigning different type
+// count = false;
+```
+
+C++ is a multi paradigm language and functions can be defined in a _class_ or outside. The [`class` keyword][class] is used to define a class. A [function][function] within a class is refered to as a [_method_][methods]. Each function can have zero or more parameters. All parameters must be explicitly typed, there is no type inference for parameters. Similarly the return type must also be made explicit. There are exceptions to the previous two rules but we'll talk about them in the future. Values are returned from functions using the [`return` keyword][return]. To allow a method to be called by code in other files, the `public` access modifier must be added.
```

Comment:
Method is not a C++ nomenclature, those are known as 'member functions': [Non-static member functions](https://en.cppreference.com/w/cpp/language/member_functions)

Reply:
Conceptually they're same thing though, if a student already knows another language wouldn't it be better to use a more familiar term?

Reply:
That is definitely the case, methods are better-known term for OOP, from the language point of view, member function is more accurate. There should be a mentioning about that similarity. However, for I believe that we should stay within definitions used by an official documentation.